src/ngx_http_lua_kong_module.c

```diff
@@ -128,7 +136,21 @@ ngx_http_lua_kong_create_loc_conf(ngx_conf_t* cf)
         return NULL;
     }
 
+    conf->error_log_append = NGX_CONF_UNSET_PTR;
+
     return conf;
 }
 
 
+static char*
+ngx_http_lua_kong_merge_loc_conf(ngx_conf_t *cf, void *parent, void *child)
+{
+    ngx_http_lua_kong_loc_conf_t *prev = parent;
+    ngx_http_lua_kong_loc_conf_t *conf = child;
+
+    ngx_conf_merge_ptr_value(conf->error_log_append, prev->error_log_append, NGX_CONF_UNSET_PTR);
+
```

Comment:
In this case the `tag` variable should be merged too, again, I think it was an overlook before.

Reply:
`tag` is `NGX_HTTP_LOC_CONF`, it will not appear in `http` or `server` scope.

Reply:
Ah, okay I didn't notice that. Maybe we should add a comment like:

```
/* conf->tag is NGX_HTTP_LOC_CONF only */
```